junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/InvocationInterceptorChain.java

```diff
@@ -137,6 +147,12 @@ private void fail(String prefix) {
 			throw new JUnitException(prefix + ": " + commaSeparatedInterceptorClasses);
 		}
 
+		@Override
+		public void skip() {
+			LOG.debug(() -> "The invocation is skipped");
+			invokedOrSkipped.set(true);
```

Comment:
For now, this should fail if `proceed()` or `skip()` have already been called, i.e. we need the same `if` as in `proceed()`.